docs/pbc/all/tax-management/202307.0/vertex/integrate-vertex.md

```diff
@@ -0,0 +1,520 @@
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return \Spryker\Zed\Kernel\Container
+     */
+    protected function extendCommandPlugins(Container $container): Container
+    {
+         $container->extend(self::COMMAND_PLUGINS, function (CommandCollectionInterface $commandCollection) {
+             
+             // ...
+             
+             $commandCollection->add(new SendPaymentTaxInvoicePlugin(), 'TaxApp/SubmitPaymentTaxInvoice');
+
+             // ...
+            
+             return $commandCollection;
+        });
+        
+        return $container;
+    }
+
+```
+
+### 4. Configure Calculation Dependency Provider
+
+Add the following to `src/Pyz/Zed/Calculation/CalculationDependencyProvider.php`
+
+```php
+// ...
+
+use Spryker\Zed\TaxApp\Communication\Plugin\Calculation\TaxAppCalculationPlugin;
+
+// ...
+
+    /**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return array<\Spryker\Zed\CalculationExtension\Dependency\Plugin\CalculationPluginInterface>
+     */
+    protected function getQuoteCalculatorPluginStack(Container $container): array
+    {
+        /** @var array<\Spryker\Zed\Calculation\Dependency\Plugin\CalculationPluginInterface> $pluginStack */
+        $pluginStack = [
+            // ...
+        
+            new TaxAppCalculationPlugin(),
+        
+            // ...
+        ];
+        
+        return $pluginStack;
+    }
+
+// ...
+```
+
+### 5. Configure Tax App Dependency Provider
+
+Tax App Dependency Provider could have next configuration `src/Pyz/Zed/TaxApp/TaxAppDependencyProvider.php`:
+
+```php
+<?php
+
+namespace Pyz\Zed\TaxApp;
+
+// ...
+
+class TaxAppDependencyProvider extends SprykerTaxAppDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Zed\TaxAppExtension\Dependency\Plugin\CalculableObjectTaxAppExpanderPluginInterface>
+     */
+    protected function getCalculableObjectExpanderPluginCollection(): array
+    {
+        return [
+            new CalculableObjectCustomerWithTaxCodeExpanderPlugin(),
+            new CalculableObjectExpensesWithTaxCodeExpanderPlugin(),
+            new CalculableObjectItemProductOptionWithTaxCodeExpanderPlugin(),
+            new CalculableObjectItemWithTaxTaxCodeExpanderPlugin(),
+        ];
+    }
+
+    /**
+     * @return array<\Spryker\Zed\TaxAppExtension\Dependency\Plugin\OrderTaxAppExpanderPluginInterface>
+     */
+    protected function getOrderExpanderPluginCollection(): array
+    {
+        return [
+            new OrderCustomerWithTaxCodeExpanderPlugin(),
+            new OrderExpensesWithTaxCodeExpanderPlugin(),
+            new OrderItemProductOptionWithTaxCodeExpanderPlugin(),
+            new OrderItemWithTaxTaxCodeExpanderPlugin(),
+        ];
+    }
+}
+```
+
+{% info_block infoBox "Note" %}
+
+Please pay attention that you have to implement that plugin stack on your side. `YourModule` - means that you'll put them wherever you suggest in some module.
+
+{% endinfo_block %}
```

Comment:
please move this box before TaxAppDependencyProvider code block.

BTW we wanted to provide `spryker-projects/tax-app-vertex` module as an example of these plugins.